frontend/src/utils/array/helpers.js

```diff
@@ -33,5 +33,8 @@ const compare = (a, b) => {
 }
 
 export const sortCategories = categories => {
-  return categories.sort(compare)
+  if (Array.isArray(categories)) {
+    return categories?.sort(compare)
+  }
+  return []
```

Comment:
Not sure if only array may be passed to this function but if so i guess that this can be:

```js
export const sortCategories = (categories = []) => categories.sort(compare)
```